LoopBack/LBUser.m

```diff
@@ -91,9 +117,35 @@ - (void)logoutWithSuccess:(LBUserLogoutSuccessBlock)success
                      success:^(id value) {
                          LBRESTAdapter* adapter = (LBRESTAdapter*)self.adapter;
                          adapter.accessToken = nil;
+                         self.currentUserId = nil;
```

Comment:
You should set `self.cachedCurrentUser = nil` as well. Please add a test (or extend an existing test) to cover this contract too.